cmd/agent/install_script.sh

```diff
@@ -142,27 +143,27 @@ elif [ $OS = "Debian" ]; then
     if [ ! -z "$cache_output" ]; then
       $sudo_cmd apt-get install -y dirmngr
     fi
-    printf "\033[34m\n* Installing APT package sources for Datadog\n\033[0m\n"
-    $sudo_cmd sh -c "echo 'deb https://apt.${dd_url}/ stable 6' > /etc/apt/sources.list.d/datadog.list"
-    $sudo_cmd apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 382E94DE
+    printf "\033[34m\n* Configuring APT package sources for StackState\n\033[0m\n"
+    $sudo_cmd sh -c "echo 'deb https://stackstate-agent-2.s3.amazonaws.com ${code_name} main' > /etc/apt/sources.list.d/stackstate.list"
+    $sudo_cmd apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 B3CC4376
 
-    printf "\033[34m\n* Installing the Datadog Agent package\n\033[0m\n"
+    printf "\033[34m\n* Installing the StackState Agent v2 package\n\033[0m\n"
     ERROR_MESSAGE="ERROR
-Failed to update the sources after adding the Datadog repository.
+Failed to update the sources after adding the StackState repository.
 This may be due to any of the configured APT sources failing -
 see the logs above to determine the cause.
-If the failing repository is Datadog, please contact Datadog support.
+If the failing repository is StackState, please contact StackState support.
 *****
 "
-    $sudo_cmd apt-get update -o Dir::Etc::sourcelist="sources.list.d/datadog.list" -o Dir::Etc::sourceparts="-" -o APT::Get::List-Cleanup="0"
+    $sudo_cmd apt-get update -o Dir::Etc::sourcelist="sources.list.d/stackstate.list" -o Dir::Etc::sourceparts="-" -o APT::Get::List-Cleanup="0"
     ERROR_MESSAGE="ERROR
-Failed to install the Datadog package, sometimes it may be
+Failed to install the StackState package, sometimes it may be
 due to another APT source failing. See the logs above to
 determine the cause.
-If the cause is unclear, please contact Datadog support.
+If the cause is unclear, please contact StackState support.
 *****
 "
-    $sudo_cmd apt-get install -y --force-yes datadog-agent
+    $sudo_cmd apt-get install -y --force-yes $PKG_NAME
     ERROR_MESSAGE=""
 elif [ $OS = "SUSE" ]; then
 
```

Comment:
We do not support sus yet, lets warn the user of this and quit

Reply:
print message and exit 1